CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for crash
 Unreleased
 ==========
 
+- Added command-line argument ``--timeout``, to configure network timeout
+  values in seconds. The default connect timeout is five seconds now,
+  the default read timeout is infinite.
```

Comment:
It is not strictly "infinite", but actually uses the system default as provided by Python's `socket` library. By default, it uses `None`, which means infinite. However, it can be changed, so this detail might want to be reflected here. WDYT?

Reply:
Done.